src/OrchardCore.Modules/OrchardCore.Search.Lucene/Migrations.cs

```diff
@@ -128,7 +128,7 @@ public int Create()
                                 }
                                 else
                                 {
-                                    if ((bool)included && !(bool)analyzed)
+                                    if ((bool)included && (analyzed == null || !(bool)analyzed))
```

Comment:
Here this is the opposite, in the above line 122 we check `if (analyzed == null)` so here we know that `analyzed` is not null and then the code could be kept as before without the null check.